PhysicsTools/PatAlgos/python/producersLayer1/jetUpdater_cfi.py

```diff
@@ -1,6 +1,6 @@
 import FWCore.ParameterSet.Config as cms
 
-patJetsUpdated = cms.EDProducer("PATJetUpdater",
+updatedPatJets = cms.EDProducer("PATJetUpdater",
```

Comment:
Is this renaming invisible to users (affects only transient objects)?
If it is not, why change the name?